virtual_ecosystem/models/animals/scaling_functions.py

```diff
@@ -268,6 +268,86 @@ def natural_mortality_scaling(mass: float, terms: tuple) -> float:
     return daily_mortality_prob
 
 
+def background_mortality(u_bg: float) -> float:
+    """Constant background rate of wastebasket mortality.
+
+    This function does nothing but return a constant at the moment.
+    I am leaving it in so there is a clear way to alter the assumptions about
+    background mortality as we move into testing and validation.
+
+    Args:
+        u_bg: The constant of background mortality [day^-1].
+
+    Returns:
+        The background rate of mortality faced by a cohort [day^-1].
+
+    """
+
+    return u_bg
+
+
+def senescence_mortality(
+    lambda_se: float, t_to_maturity: float, t_since_maturity: float
+) -> float:
+    """Age-based mortality.
+
+    Madingley describes the equation as exp(time_to_maturity/time_since_maturity) but I
+    suspect this is an error and that it should be inverted. If, for example, it took
+    1000 days to reach maturity and the cohort had been mature for 1 day, then the
+    instantaneous rate of senescence mortality would be lambda_se * exp(1000/1). This
+    would also mean that the rate of senescence would decrease over time. Therefore, I
+    have inverted the relationship below.
+
+    Args:
+        lambda_se: The instantaneous rate of senescence mortality at point of maturity
+                    [day^-1].
+        t_to_maturity: The time it took the cohort to reach maturity [days].
+        t_since_maturity: The time elapsed since the cohort reached maturity [days].
+
+    Returns:
+        The rate of senescence mortality faced by an animal cohort [day^-1].
+
+    """
+
+    t_pm = t_to_maturity  # time it took to reach maturity
+    t_bm = t_since_maturity  # time since maturity
+
+    u_se = lambda_se * exp(t_bm / t_pm)
+
+    return u_se
+
+
+def starvation_mortality(
+    lambda_max: float, J_st: float, zeta_st: float, mass_current: float, mass_max: float
+) -> float:
+    """Mortality from body-mass loss.
+
+    There is a error in the madingley paper that does not follow their source code. The
```

Comment:
The paper seems to be full of errors 😄

Reply:
There have been some unpleasant surprises! When I get to doing the final parametrization, I'm going to comb through all of their actual C# code to make sure.

Reply:
It should be them publishing an addendum...